src/components/CuratorDashboard/index.tsx

```
@@ -0,0 +1,125 @@
+const CuratorDashboard = ({ className }: Props) => {
+	const { resolvedTheme: theme } = useTheme();
+	const { id, loginAddress } = useUserDetailsSelector();
+	const [openLoginPrompt, setOpenLoginPrompt] = useState<boolean>(false);
+	const [openAddressLinkedModal, setOpenAddressLinkedModal] = useState<boolean>(false);
+	const [openModal, setOpenModal] = useState<boolean>(false);
+	const [proposerAddress, setProposerAddress] = useState<string>('');
+	const [openLogin, setLoginOpen] = useState<boolean>(false);
+	const [openSignup, setSignupOpen] = useState<boolean>(false);
+
+	const handleClick = () => {
+		if (id && !isNaN(id)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)